### DISCUSSION TITLE
Example of elegant list filtering?

### DISCUSSION THREAD

**Discussion by BenJackGill** [quasarframework/quasar#14612] [General - Components / Directives / etc]
I have a list of expansion items, and I would like to allow the user to filter the list.

But I'm not sure of the best to way to implement that.

Was hoping to get some inspiration from the community.

What UI elements would you use to filter a large list?

**Comment by metalsadman** [comment]
expansion items is ok, the filtering can be done using q-input. assuming you build your items with an array of objects, you filter that in the qinput handler, if it's deeply nested then you call it recursively.
